packages/button-group/src/index.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import {Button} from '@smashing/button'
+import {ButtonGroupProps, Option} from './types'
+
+const StyledButton = styled(Button)<{checked?: boolean}>`
+  border-radius: 0;
+  ${_ =>
+    _.checked && {
+      backgroundColor: _.theme.scales.blue.B3A,
+      backgroundImage: 'none'
+    }}
+  &:first-child {
+    border-radius: ${_ => `${_.theme.radius} 0 0 ${_.theme.radius}`};
+  }
+  &:last-child {
+    border-radius: ${_ => `0 ${_.theme.radius} ${_.theme.radius} 0`};
```

Comment:
Make sure that there is >one< pixel border between buttons - apparently there are two right now.